winterfell/webapp/src/js/profileService.js

```diff
@@ -34,8 +34,9 @@ app.service('Profile', ['claimService', '$q', 'net', function(claimService, $q,
 
       var that = this;
       net.getUserInfo().then(function success(res) {
-        that.setUser(res.data);
```

Comment:
Ah I see this is because of the "dev mode" XHR for httpGet. Is that still necessary? When do you have a need to test things without a server backend?